graviti/operation/frame.py

```diff
@@ -134,3 +163,29 @@ def do(  # pylint: disable=invalid-name
                 sheet=sheet,
                 data=batch,
             )
+
+
+def _get_file_arrays(schema: record, dataframe: "DataFrame") -> List["Container"]:
+    arrays = []
+    for key in _get_file_column_keys(schema):
+        array: "Container" = dataframe
+        for subkey in key:
+            array = array[subkey]  # type: ignore[index]
+        arrays.append(array)
+    return arrays
+
+
+def _get_file_column_keys(schema: record) -> List[Tuple[str, ...]]:
```

Comment:
Shall we make this function more common, make it to search any type not only File from schema?